packages/node/src/types.ts

```diff
@@ -56,4 +56,6 @@ export interface NodeOptions extends Options<NodeTransportOptions>, BaseNodeOpti
  * Configuration options for the Sentry Node SDK Client class
  * @see NodeClient for more information.
  */
-export interface NodeClientOptions extends ClientOptions<NodeTransportOptions>, BaseNodeOptions {}
+export interface NodeClientOptions extends ClientOptions<NodeTransportOptions>, BaseNodeOptions {
+  profilesSampleRate?: number;
```

Comment:
Can we add a JSDoc here, explaining what this option does?

Reply:
Good call, I added it